### ISSUE TITLE
Support more private queues

### ISSUE THREAD

**Issue by jonespm** [tl-its-umich-edu/remote-office-hours-queue#70]
I'm not sure if it's come up but it seems like having private queues that don't come up in the search and are more difficult to discover might be preferred. It seems like the queues are autonumbered and all in the search now which makes discovering any queues relatively easy. I could see a someone wanting a queue that wasn't searchable and maybe had a less guessable ID (like a GUID). Maybe this is two tickets and it would be worthwhile to have all ids be GUID's.

**Comment by mfldavidson** [comment]
I think this is a duplicate of #20 ?

**Comment by jonespm** [comment]
Yeah sounds similar to that one, I think this one has a little more details on how this might be implemented. I didn't see that when looking through the list.

**Comment by mfldavidson** [comment]
I'm going to fold #20 into this one.

Note from #20: This was noted as high priority by an advisor beta tester
